recipe.py

```diff
@@ -32,21 +32,19 @@ def set_calories_per_portion(self, calories_per_portion:int):
     def get_ingredients(self):
         return self._ingredients
     
-    def set_ingredients(self,ingredients):
+    def set_ingredients(self,ingredients:list):
+        self._ingredients = ingredients
+
+    def add_ingredients(self,ingredients):
         self._ingredients.append(ingredients)
-        if len(set(self._ingredients)) != len(self._ingredients):
-            self.remove_ingredient(ingredients) #I have tested this and it only removes the recently added ingredient
-            return False
+        # if len(set(self._ingredients)) != len(self._ingredients):
+        #     self.remove_ingredient(ingredients) #I have tested this and it only removes the recently added ingredient
+        #     return False
     
     def remove_ingredient(self,ingredient):
         self._ingredients.remove(ingredient)
         #[x for x in self._ingredients if x is not None] #tried to use list comprehension to get rid of the none value in self._ingredients did not succeed
         #Do I need to prevent this removing duplicates if I have already prevented duplication with the set ingredients validation
```

Comment:
Arguably the ingredients should be unique, but at this point a degree of permissiveness is required to progress things.